include/linux/bpf.h

```diff
@@ -29,3 +30,131 @@ enum bpf_stats_type
 {
     BPF_STATS_TYPE_UNKNOWN
 };
+
+enum bpf_cmd_id
+{
+    BPF_MAP_CREATE,
+    BPF_MAP_LOOKUP_ELEM,
+    BPF_MAP_UPDATE_ELEM,
+    BPF_MAP_DELETE_ELEM,
+    BPF_MAP_GET_NEXT_KEY,
+    BPF_PROG_LOAD,
+    BPF_OBJ_PIN,
+    BPF_OBJ_GET,
+    BPF_PROG_GET_NEXT_ID,
+    BPF_MAP_GET_NEXT_ID,
+    BPF_LINK_GET_NEXT_ID,
+    BPF_PROG_GET_FD_BY_ID,
+    BPF_MAP_GET_FD_BY_ID,
+    BPF_LINK_GET_FD_BY_ID,
+    BPF_OBJ_GET_INFO_BY_FD,
+    BPF_LINK_DETACH,
+    BPF_PROG_BIND_MAP,
+};
+
+/// Attributes used by BPF_OBJ_GET_INFO_BY_FD.
+typedef struct
+{
+    uint32_t bpf_fd;   ///< File descriptor referring to an eBPF object.
```

Comment:
nit: is bpf_ required ?

Reply:
if it is along below of link_, prog_, would suggest a specific prefix be used

Reply:
The field names must match what Linux etc has so code can be portable.